pygmt/datasets/earth_free_air_anomaly.py

```diff
@@ -63,44 +70,58 @@ def load_earth_free_air_anomaly(
     registration
         Grid registration type. Either ``"pixel"`` for pixel registration or
         ``"gridline"`` for gridline registration. Default is ``None``, means
-        ``"gridline"`` for all resolutions except ``"01m"`` which is
-        ``"pixel"`` only.
+        ``"gridline"`` for all resolutions except ``"01m"`` which is ``"pixel"`` only.
+    Uncertainty
+        Select the free air anomaly data. Available options are:
+
+        - ``"faa"``: Altimetry-based marine free-air anomaly values.
+        - ``"faaerror"``: Uncertainties to the free-air anomaly values.
 
     Returns
     -------
     grid
-        The Earth free-air anomaly grid. Coordinates are latitude and
-        longitude in degrees. Units are in mGal.
+        The Earth free-air anomaly grid. Coordinates are latitude and longitude in
+        degrees Units are in mGal.
 
     Note
     ----
     The registration and coordinate system type of the returned
-    :class:`xarray.DataArray` grid can be accessed via the GMT accessors
-    (i.e., ``grid.gmt.registration`` and ``grid.gmt.gtype`` respectively).
-    However, these properties may be lost after specific grid operations (such
-    as slicing) and will need to be manually set before passing the grid to any
-    PyGMT data processing or plotting functions. Refer to
-    :class:`pygmt.GMTDataArrayAccessor` for detailed explanations and
-    workarounds.
+    :class:`xarray.DataArray` grid can be accessed via the GMT accessors (i.e.,
+    ``grid.gmt.registration`` and ``grid.gmt.gtype`` respectively). However, these
+    properties may be lost after specific grid operations (such as slicing) and will
+    need to be manually set before passing the grid to any PyGMT data processing or
+    plotting functions. Refer to :class:`pygmt.GMTDataArrayAccessor` for detailed
+    explanations and workarounds.
 
     Examples
     --------
 
     >>> from pygmt.datasets import load_earth_free_air_anomaly
     >>> # load the default grid (gridline-registered 1 arc-degree grid)
     >>> grid = load_earth_free_air_anomaly()
+    >>> # load the uncertainties related to the default grid
+    >>> grid = load_earth_free_air_anomaly(uncertainty=True)
     >>> # load the 30 arc-minutes grid with "gridline" registration
     >>> grid = load_earth_free_air_anomaly(resolution="30m", registration="gridline")
     >>> # load high-resolution (5 arc-minutes) grid for a specific region
     >>> grid = load_earth_free_air_anomaly(
-    ...     resolution="05m",
-    ...     region=[120, 160, 30, 60],
-    ...     registration="gridline",
+    ...     resolution="05m", region=[120, 160, 30, 60], registration="gridline"
     ... )
     """
+    # Map data source to prefix
+    prefix = {
+        False: "earth_faa",
+        True: "earth_faaerror",
+    }.get(uncertainty)
```

Comment:
```suggestion
    prefix = "earth_faaerror" if prefix is True else "earth_faa"
```

Reply:
Changed this in commit https://github.com/GenericMappingTools/pygmt/pull/3727/commits/8245f6d7ded5df7d8a1524d4847e5dba3a635387.